.github/workflows/ci.yml

```diff
@@ -25,13 +25,13 @@ jobs:
         os: [ubuntu-latest, macos-latest, windows-latest]
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
```

Comment:
I've recently tried dependabot against ci.yml in one of my personal projects to automatically update dependencies like these. It worked very well, may be we should enable it against some containerd's repos.

Reply:
sure, we can try it again